x-pack/plugins/log_explorer/public/customizations/custom_flyout_content.tsx

```diff
@@ -5,23 +5,40 @@
  * 2.0.
  */
 
-import React from 'react';
+import React, { useCallback } from 'react';
 import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
 import { FlyoutDetail } from '../components/flyout_detail/flyout_detail';
 import { FlyoutProps } from '../components/flyout_detail';
+import { useLogExplorerCustomizationsContext } from '../hooks/use_log_explorer_customizations';
 
 export const CustomFlyoutContent = ({
   actions,
   dataView,
   doc,
   renderDefaultContent,
 }: FlyoutProps) => {
+  const { flyout } = useLogExplorerCustomizationsContext();
+
+  const renderPreviousContent = useCallback(
+    () => (
+      <>
+        {/* Apply custom Log Explorer detail */}
+        <EuiFlexItem>
+          <FlyoutDetail actions={actions} dataView={dataView} doc={doc} />
+        </EuiFlexItem>
+      </>
+    ),
+    [actions, dataView, doc]
+  );
+
+  const content = flyout?.renderContent
```

Comment:
So here you will be able to check as to if `flyout.customContent` exists, then render custom component with previousContent